src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -39,8 +46,13 @@
     public static final String VALID_TAG_HUSBAND = "husband";
     public static final String VALID_TAG_FRIEND = "friend";
     public static final String VALID_DATE_OCTOBER = "2020-10-10 10:00";
+    public static final String VALID_DATE_NOVEMBER = "2020-11-11 11:00";
     public static final String VALID_TITLE_MEETING = "Meeting";
+    public static final String VALID_TITLE_MOVIE = "Movie";
     public static final String VALID_DESCRIPTION_STORY = "Brainstorm user stories";
+    public static final String VALID_DESCRIPTION_MOVIE = "Watch a movie";
+    public static final String CONTACT_DEFAULT_UUID = "e26616c9-c740-4d86"
+            + "-861e-733a4d377a3e";
```

Comment:
This is the same UUID as used in ParserUtil's parseContacts. Again, it's just a placeholder that's used to make sure the equals method doesn't just break.